frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignTab.tsx

```diff
@@ -75,12 +86,50 @@ export const BuilderAndDesignTab = (): JSX.Element => {
             destination.index,
           )
         }
+
         case CREATE_FIELD_DROP_ID: {
           return setToCreating(
             getFieldCreationMeta(CREATE_FIELD_FIELDS_ORDERED[source.index]),
             destination.index,
           )
         }
+
+        case CREATE_MYINFO_PERSONAL_DROP_ID: {
+          return setToCreating(
+            getMyInfoFieldCreationMeta(
+              CREATE_MYINFO_PERSONAL_FIELDS_ORDERED[source.index],
+            ),
+            destination.index,
+          )
+        }
+
+        case CREATE_MYINFO_CONTACT_DROP_ID: {
+          return setToCreating(
+            getMyInfoFieldCreationMeta(
+              CREATE_MYINFO_CONTACT_FIELDS_ORDERED[source.index],
+            ),
+            destination.index,
+          )
+        }
+
+        case CREATE_MYINFO_PARTICULARS_DROP_ID: {
+          return setToCreating(
+            getMyInfoFieldCreationMeta(
+              CREATE_MYINFO_PARTICULARS_FIELDS_ORDERED[source.index],
+            ),
+            destination.index,
+          )
+        }
+
+        case CREATE_MYINFO_MARRIAGE_DROP_ID: {
+          return setToCreating(
+            getMyInfoFieldCreationMeta(
+              CREATE_MYINFO_MARRIAGE_FIELDS_ORDERED[source.index],
+            ),
+            destination.index,
+          )
+        }
```

Comment:
do we actually need all these separate ids? can we not group them in one like how we do it for basic fields?

Reply:
sorry, could i get some more clarity on this ah cos i think it's implemented in a similar way ._.

Reply:
oh nevermind, it is indeed implemented in a similar way in basicfields haha. ignore me